components/docs-chef-io/content/habitat/origin-settings.md

```diff
@@ -0,0 +1,105 @@
++++
+title = "Origin Settings"
+
+date = 2020-10-12T14:02:01-07:00
+draft = false
+
+[menu]
+  [menu.habitat]
+    title = "Origin Settings"
+    identifier = "habitat/origins/origin-settings Origin Settings"
+    parent = "habitat/origins"
+    weight = 40
++++
+
+[\[edit on GitHub\]](https://github.com/habitat-sh/habitat/blob/master/components/docs-chef-io/content/habitat/origin-settings.md)
+
+The _Origin Settings_ tab contains:
+
+* Default Package Settings
+* Origin Secrets
+
+Everyone with origin membership can see the _Settings_ tab, but only origin administrators and owners can add, update, or delete settings content.
+
+| Settings Actions | Read-Only | Member | Maintainer | Administrator | Owner |
+|---------|-------|-------|-------|-------|-------|
+| View settings | Y | Y | Y | Y | Y |
+| Add/Update/Delete settings | N | N | N | Y | Y |
+| **Origin Secrets Actions** |
+| View secrets | N | N | Y | Y | Y |
+| Add/Update/Delete secrets | N | N | N | Y | Y |
```

Comment:
Is there any magic that allows these ASCII tables to be manually aligned in the source, but still generate the proper tables in the rendered output?

When reviewing, it would make it much easier to visually verify the accuracy of the table without having to do all the row/column bookkeeping mentally.

Reply:
Sadly, it would need to be done manually for this table, because it is kind of complex.

Reply:
HOWEVER, you can preview it on the netlify build